streaming/emr-eks-flink/examples/karpenter/basic-example-app-cluster.yaml

```diff
@@ -26,8 +26,8 @@ spec:
     state.checkpoints.dir: s3://emr-flink-data/checkpoints
     state.savepoints.dir: s3://emr-flink-data/savepoints
 
-
-  executionRoleArn: arn:aws:iam::xxxxxxxxx:role/emr-eks-flink-flink-team-a-20240406012025932700000008
+  # Replace this execution role ARN with your own
+  executionRoleArn: arn:aws:iam::681921237057:role/emr-eks-flink-flink-team-a-20240406012025932700000008
```

Comment:
We should substitute this with variables, or provide instructions on how to generate the role arn (including the permissions).